bigtable/bigtable.go

```diff
@@ -619,6 +621,14 @@ func (m *Mutation) Set(family, column string, ts Timestamp, value []byte) {
 	}}})
 }
 
+// SetInt64 sets a value in a specified column, with the given timestamp.
+// It performs the 64-bit big-endian signed integer encoding.
+func (m *Mutation) SetInt64(family, column string, ts Timestamp, value int) {
```

Comment:
Seems like this should be an `int64` argument if we always treat it as an `int64`, especially given the name of the method?